packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -1432,111 +1240,82 @@ export namespace EmrCreateCluster {
     /**
      * The identifier of the Amazon EC2 security group for the master node.
      *
-     * @default No emrManagedMasterSecurityGroup
+     * @default - None
      */
     readonly emrManagedMasterSecurityGroup?: string;
 
     /**
      * The identifier of the Amazon EC2 security group for the core and task nodes.
      *
-     * @default No emrManagedSlaveSecurityGroup
+     * @default - None
      */
     readonly emrManagedSlaveSecurityGroup?: string;
 
     /**
      * Applies only to Amazon EMR release versions earlier than 4.0. The Hadoop version for the cluster.
      *
-     * @default No hadoopVersion
+     * @default - None
```

Comment:
What does none mean here? It has to pick some version right?

Reply:
updated to describe the different versions that can be present and the scenarios where they are